lessons/basics/pipe-operator.md

```diff
@@ -53,18 +53,17 @@ true
 
 ## Best Practices
 
-If the arity of a function is more than 1, then make sure to use parenthesis. This doesn't matter much to the Elixir, but it matters to other programmers who may misinterpret your code. If we take our 2nd example, and remove the brackets from `Enum.map/2`, we are met with the following warning.
+If the arity of a function is more than 1, then make sure to use parenthesis. This doesn't matter much to the Elixir, but it matters to other programmers who may misinterpret your code. If we take our 3rd example, and remove the brackets from `String.ends_with?/2`, we are met with the following warning.
```

Comment:
`parenthesis` should be `parentheses`. I wasnt sure but checked - [source](http://www.dictionary.com/browse/parentheses)